components/projectPage/languages/Languages.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import style from './Languages.module.scss';
+
+function Languages() {
+  const languages = ['HTML', 'CSS', 'JavaScript']; //in the future: API
+  const languagesArr = languages.map((language) => {
+    return (
+      <li key={language} className={style.languages}>
```

Comment:
this is short function, I think it will be better in this used to not separate it from the ul context